### PR TITLE
Feature add entity

### PR DESCRIPTION
**Pull request by maksim-luzin**
Added tables for the backend according to the scheme and taking into account the wishes.

### CODE REVIEW

File: backend/src/data/entities/project.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  @Column()
+  finishDate: Date;
+
+  @Column({ type: 'int' })
+  totalViews: number;
+
+  @Column({ type: 'int' })
+  minutesToRead: number;
+
+  @Column({ type: 'int' })
```
Comment (yevhenii-moroziuk) [inline backend/src/data/entities/project.ts]: This field should be real

File: backend/src/data/entities/fAQ.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Entity, Column, ManyToOne } from 'typeorm';
```
Comment (mtseluiko) [inline backend/src/data/entities/fAQ.ts]: Rename this file to faq

File: backend/src/data/entities/achievement.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  description: string;
+
+  @Column({ type: 'money' })
+  goal: number;
+
+  @OneToMany(
+    () => UserAchievement,
+    userAchievement => userAchievement.achievement
+  )
+  userAchievements!: UserAchievement[];
```
Comment (yevhenii-moroziuk) [inline backend/src/data/entities/achievement.ts]: Why do we need exclamation point?

File: backend/src/data/entities/fAQ.ts (hunk 2 above)
Comment (yevhenii-moroziuk) [inline backend/src/data/entities/fAQ.ts:1]: Let it be faq.ts

**Review by yevhenii-moroziuk — CHANGES_REQUESTED** [review]

File: backend/src/data/entities/fAQ.ts (hunk 2 above)
Comment (maksim-luzin) [reply backend/src/data/entities/fAQ.ts]: I renamed.

File: backend/src/data/entities/project.ts (hunk 1 above)
Comment (maksim-luzin) [reply backend/src/data/entities/project.ts]: I did it real.

File: backend/src/data/entities/achievement.ts (hunk 3 above)
Comment (maksim-luzin) [reply backend/src/data/entities/achievement.ts]: I deleted exclamation point.

File: backend/src/data/entities/project.ts (hunk 1 above)
Comment (yevhenii-moroziuk) [reply backend/src/data/entities/project.ts]: Have you changed totalInteractionTime? It seems no.

**Review by yevhenii-moroziuk — APPROVED** [review]

**Pull request merged**